kentikapi/internal/httputil/httputil.go

```diff
@@ -61,28 +57,12 @@ type RetryConfig struct {
 	MinDelay *time.Duration
 	// MaxDelay is a maximum delay before request retry. Default: 30 seconds.
 	MaxDelay *time.Duration
-	// RetryableStatusCodes are HTTP response status codes to retry on. Default: [429, 500, 502, 503, 504].
-	RetryableStatusCodes []int
-	// RetryableMethods are HTTP request retry methods, which the retry strategy is enabled for.
-	// Default: [GET, HEAD, POST, PUT, PATCH, DELETE, CONNECT, OPTIONS, TRACE].
-	RetryableMethods []string
 }
 
 func (cfg *ClientConfig) FillDefaults() {
 	if cfg.HTTPClient == nil {
 		cfg.HTTPClient = defaultHTTPClient()
 	}
```

Comment:
Please call cfg.RetryConfig.FillDefaults() here so that httputil.NewRetryingClient() has safe interface and does not panic when empty config is passed

Reply:
Please move `cfg.RetryCfg.FillDefaults()` call to the body of `cfg.FillDefaults()`  (i.e. `ClientConfig.FillFefaults()`). That way filling defaults is in one place (at the beggining of `NewRetryingClient()`) and is before initializations.

Reply:
Done